lib/ansible/module_utils/vmware.py

```diff
@@ -806,7 +809,7 @@ def __init__(self, module):
         self.params = module.params
         self.si = None
         self.current_vm_obj = None
-        self.content = connect_to_api(self.module)
+        self.si, self.content = connect_to_api(self.module)
```

Comment:
Don't you need to call connect_to_api with `return_si` to `True`?

```suggestion
        self.si, self.content = connect_to_api(self.module,return_si=True)
```

Reply:
Yep

Reply:
Copied file from ansible main repo, so I guess we can resolve this conversation